shtdlib.sh

```diff
@@ -1608,6 +1614,7 @@ function test_stdlib()
     for ((i=1; i <= 11 ; i++)) ; do
         debug ${i} "Debug Level ${i}"
     done
+    verbosity="${old_verbosity}"
```

Comment:
Should this variable be 'orig_verbosity'?

Reply:
Sorry, forgot to push latest